### PR TITLE
Add new Beam Java SDK examples

### PR DESCRIPTION
**Pull request by TSultanov**
These examples are intended for use in interactive embedded Playground frames in the Beam documentation.

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: learning/beamdoc/java/GroupIntoBatches.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+//   name: GroupIntoBatches
+//   description: Demonstration of GroupIntoBatches transform usage.
+//   multifile: false
+//   default_example: false
+//   context_line: 26
+//   categories:
+//     - Core Transforms
+//   complexity: BASIC
+//   tags:
+//     - transform
```
Comment (rshamunov) [inline learning/beamdoc/java/GroupIntoBatches.java:20]: Use "transforms" (plural) tag

**Comment by TSultanov** [comment]
Run Java PreCommit

**Comment by TSultanov** [comment]
Run Java_hadoop_IO_Direct PreCommit

**Comment by TSultanov** [comment]
Run Java PreCommit

File: examples/java/src/main/java/org/apache/beam/examples/CombineExample.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+import org.slf4j.LoggerFactory;
+
+// beam-playground:
+//   name: Combine
+//   description: Demonstration of Combine transform usage.
+//   multifile: false
+//   default_example: false
+//   context_line: 46
+//   categories:
+//     - Core Transforms
```
Comment (rshamunov) [inline examples/java/src/main/java/org/apache/beam/examples/CombineExample.java:39]: - Combiners

**Comment by pabloem** [comment]
lgtm

**Pull request merged**